### DISCUSSION TITLE
请问如何让newbing看一些长的文本或者图片？

### DISCUSSION THREAD

**Discussion by falsemaskx** [viopsa233/AI-testers-Forum#58] [Q&A]
请问如何让newbing看一些长的文本或者图片？即使按他的要求做，最后也往往只会胡言乱语

**Comment by q2408808** [comment]
看不了过长的文本，限制也就1万多token 3万左右的单词或者文字。图片好像也不行吧 暂时还不支持多模态输入

**Comment by falsemaskx** [comment]
主要是，他会建议我给他发链接，但是即使发给他了，他也看不到，只会胡言乱语，有什么解决方案吗？还是确实不能看链接？

**Reply by ghost** [reply]
看不了
